JokeRandomizer/JokeRandomizer/Views/FirstScreenViewController.swift

```diff
@@ -6,11 +6,19 @@
 //
 
 import UIKit
+import Combine
+import CoreData
 
-class FirstScreen: UIViewController {
+class FirstScreenViewController: UIViewController {
 //MARK: - Property
     private let jokesVM = JokesViewModel()
```

Comment:
don't initialize the viewModel here. use dependency injection

Reply:
will check for the dependency injection on how to apply it.